### ISSUE TITLE
Desktop application?

### ISSUE THREAD

**Issue by OffifialMITX** [LonamiWebs/Stringlate#66]
`Some applications have a large amount of strings waiting to be translated. This is the reason why it's not a desktop application, so it could be done anywhere (no internet connection is required once the strings are saved).`

I could understand if you said you just don't have the time but many people can write much faster on a physical keyboard and I think you know this.

In my opinion a desktop application would be very nice (even more nice with the ability to sync and even much more nice if via Nextcloud)

I'll close the issue because it's not a technical discussion.

**Comment by XyLoNaMiyX** [comment]
> I could understand if you said you just don't have the time but many people can write much faster on a physical keyboard and I think you know this.

Yes I know, but in my case, there are some moments in which I can't use my laptop but have nothing better to do, and so other people might be in the same situation. I decided to make this app even though I already did the translation of what I needed because it was a good idea.

> In my opinion a desktop application would be very nice (even more nice with the ability to sync […])

Yes, it would be cool. But there are more collaborative projects online, aren't there? Like Transifex and such.

> I'll close the issue because it's not a technical discussion.

You can keep it open for discussion, and maybe someone else could see this and do it ;)

**Comment by OffifialMITX** [comment]
None of the other translators are avalible as a FOSS offline app (yet) so it would make your app much more popular if it was avalivle for desktop PCs either (at least that's what I think).

**Comment by XyLoNaMiyX** [comment]
> None of the other translators are avalible as a FOSS offline app

And the part that most annoys me is that they need you to *log in*. Sure, they're online platforms and they need some kind of authentification… But I find it a bit annoying that you need to make yet another account only to translate one application and maybe never use the platform again.

**Comment by licaon-kter** [comment]
Transifex, Crowdin and Weblate can auth via Github, no account needed.

**Comment by XyLoNaMiyX** [comment]
> Transifex, Crowdin and Weblate can auth via Github, no account needed.

Didn't know that, I usually prefer to make an account per site so probably missed it…

**Comment by OffifialMITX** [comment]
I did know it but I prefer to make an extra account antway - big data.

Am 7. Januar 2017 12:32:44 MEZ, schrieb Lonami <notifications@github.com>:
>> Transifex, Crowdin and Weblate can auth via Github, no account
>needed.
>
>Didn't know that, I usually prefer to make an account per site so
>probably missed it…
>
>-- 
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/LonamiWebs/Stringlate/issues/66#issuecomment-271078277

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.
